### PR TITLE
FHAC-558 PR.  Removed ParticipantObjectName from PD,QD and RD

### PR DESCRIPTION
**Pull request by TabassumJafri**
FHAC-558: Added fix for ParticipantObjectName in PD, QD,RD and updated interface in X12.

### CODE REVIEW

**Comment by christophermay07** [comment]
Question -- you have several added imports that you don't use. Can these be removed? (if so, make sure you have our NetBeans formatter imported, and that the option to remove unused imports on save is checked).

**Comment by christophermay07** [comment]
Question -- you have several added imports that you don't use. Can these be removed? (if so, make sure you have our NetBeans formatter imported, and that the option to remove unused imports on save is checked).

**Comment by christophermay07** [comment]
Question -- you have several added imports that you don't use. Can these be removed? (if so, make sure you have our NetBeans formatter imported, and that the option to remove unused imports on save is checked).

File: Product/Production/Services/DocumentQueryCore/src/test/java/gov/hhs/fha/nhinc/docquery/audit/transform/DocQueryAuditTransformsTest.java [hunk 1]
```diff
@@ -167,6 +170,12 @@ private void assertParticiopantObjectIdentification(LogEventRequestType auditReq
         assertEquals(DocQueryAuditTransformsConstants.PARTICIPANT_QUERY_OBJ_ID_TYPE_DISPLAY_NAME,
             auditRequest.getAuditMessage().getParticipantObjectIdentification().get(1).
             getParticipantObjectIDTypeCode().getDisplayName());
+        assertNull(null, auditRequest.getAuditMessage().getParticipantObjectIdentification().get(0).
```
Comment (christophermay07) [inline Product/Production/Services/DocumentQueryCore/src/test/java/gov/hhs/fha/nhinc/docquery/audit/transform/DocQueryAuditTransformsTest.java:173]: Don't use null as first parameter -- either make a proper assertion failure message (e.g., something like "ParticipantObjectName must be null"), or use the single-arg version of assertNull.

http://junit.org/apidocs/org/junit/Assert.html#assertNull(java.lang.String,%20java.lang.Object)
http://junit.org/apidocs/org/junit/Assert.html#assertNull(java.lang.Object)

**Comment by TabassumJafri** [comment]
Closing PR due to merge conflicts.

**Pull request closed without merging**